### PR TITLE
Customising time picker sheet + return trip booking option

### PR DESCRIPTION
**Pull request by pnuts2**
Redmine: https://redmine.buzzhives.com/issues/15500 + https://redmine.buzzhives.com/issues/15741

### CODE REVIEW

File: Sources/TripKitUI/views/results/TKUITimePickerSheet+Configuration.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+//
+//  Created by Brian Huang on 3/11/21.
+//  Copyright © 2021 SkedGo Pty Ltd. All rights reserved.
+//
+
+import Foundation
+
+public extension TKUITimePickerSheet {
+  
+  struct Configuration {
```
Comment (nighthawk) [inline Sources/TripKitUI/views/results/TKUITimePickerSheet+Configuration.swift:13]: I'll remove this and instead make this part of the configuration for the `TKUIRoutingResultCard`. The reason for that is that it doesn't make as much sense to apply this to _every_ time picker, regardless of how it's used, and it makes more sense as a configuration of whichever use case this is being used in.

File: Sources/TripKit/core/Loc+TripKitUI.swift [hunk 2]
```diff
@@ -77,6 +77,10 @@ extension Loc {
   public static var EndLocation: String {
     return NSLocalizedString("End location", tableName: "TripKit", bundle: .tripKit, comment: "Placeholder name for destination (then replaced with address or name)")
   }
+  
+  public static var SetTime: String {
+    return NSLocalizedString("Set time", tableName: "TripKit", bundle: .tripKit, comment: "Button label tos et departure time")
```
Comment (pnuts2) [inline Sources/TripKit/core/Loc+TripKitUI.swift:82]: Typo

**Review by pnuts2 — COMMENTED** [review]
LGTM, will do some testing now.

File: Sources/TripKit/core/Loc+TripKitUI.swift (hunk 2 above)
Comment (nighthawk) [reply Sources/TripKit/core/Loc+TripKitUI.swift:86]: Thanks

```suggestion
    return NSLocalizedString("Set time", tableName: "TripKit", bundle: .tripKit, comment: "Button label to set departure time")
```

**Review by nighthawk — APPROVED** [review]
All good from my end

**Pull request merged**